src/config/dynamic-theme-fixes.config

```diff
@@ -7978,6 +7978,8 @@ img[src="assets/Uploads/dlp/images/logo-libreoffice.png"]
 ================================
 
 docusign.com
+apps.docusign.com
+account.docusign.com
```

Comment:
Maybe use `*.docusign.com` instead of adding all found subpage?

Reply:
I did that in #12706 but it was removed ._.

I'll add it back.